binaries/hssi/ethernet/hssicommon.py

```diff
@@ -767,6 +1045,71 @@ def __init__(self):
         self.pyopaeuio_inst = pyopaeuio()
         self.num_uio_regions = 0
         self.region_index = 0
+        self.hssi_csr = 0
+
+    def verify_hssi_dfh_ver(self):
+        hssi_dfh = dfh(self.read64(0, 0))
+        print("{0: <24}:{1}".format("DFH", hex(self.read64(0, 0))))
+        print("{0: <24}:{1}".format("HSSI ID", hex(hssi_dfh.id)))
+
+        if (hssi_dfh.feature_rev == 0x1) or (hssi_dfh.feature_rev == 0):
+            print("{0: <24}:{1}".format("DFHv", 0))
+            self.hssi_csr = HSSI_DFHV0_CSR()
+        elif (hssi_dfh.feature_rev == 0x2):
+            self.hssi_csr = HSSI_DFHV05_CSR()
+            guidl = self.read64(0, self.hssi_csr.HSSI_GUID_L)
+            guidh = self.read64(0, self.hssi_csr.HSSI_GUID_H)
+            print("{0: <24}:{1}".format("DFHv", 0.5))
+            print("{0: <24}:{1}".format("guidl", hex(guidl)))
+            print("{0: <24}:{1}".format("guidh", hex(guidh)))
+
+            if guidl != self.hssi_csr.HSSI_DFHV05_GUILDL:
+                print("bad guidh {0}!={1}".format(hex(guidl),
```

Comment:
You're printing bad guidh, but the variable is hex(guidl).

Reply:
fixed print function with "bad guidl"